space-template.yaml

```diff
@@ -0,0 +1,172 @@
+---
+# These define the potential field types that ca be used in the fields of work
+# item types.
+field_types:
+  - &string string
+  - &translatable_string translatable_string
+  - &list list
+  - &timestamp timestamp
+  - &user user
+
```

Comment:
Are these base types or extensions to work items - can someone add to these? Is this something you intend to have an administrator or program manager change, add to or modify?

Does the comment need to say, supported types?